### PR TITLE
Update motion.update: Add attachment_ids and workflow code.

### PR DESCRIPTION
**Pull request by reiterl**
Add this code from motion.update_metadata.
Also update and add tests.

**Notice**: The origin field stays out. I'm not sure about that field.

### CODE REVIEW

File: tests/system/action/motion/test_update.py [hunk 1]
```diff
@@ -29,6 +29,8 @@ def test_update_correct(self) -> None:
                             "reason": "reason_ukWqADfE",
                             "modified_final_version": "mfv_ilVvBsUi",
                             "amendment_paragraph_$": {3: "<html>test</html>"},
+                            "attachment_ids": [],
+                            "supporter_ids": [],
```
Comment (jsangmeister) [inline tests/system/action/motion/test_update.py:33]: Please also add a test where you set these fields.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Comment by jsangmeister** [comment]
Also, `origin_id` has to be handled the same way as in `motion.create` - you can reuse the check from that action.

**Comment by FinnStutzenstein** [comment]
Please first wait for the discussion in https://github.com/OpenSlides/openslides-backend-coordination/issues/25

**Comment by jsangmeister** [comment]
See https://github.com/OpenSlides/openslides-backend/issues/487: please merge this PR with #448 and fuse the actions together.

**Comment by reiterl** [comment]
motion update and update_metadata are merged in another pr.

**Pull request closed without merging**